### DISCUSSION TITLE
What is Dbeaver for? (Explain to me like im 5)

### DISCUSSION THREAD

**Discussion by akhibhrat58** [dbeaver/dbeaver#10745] [Q&A] [answered]
I tried to do a search on this but it searching for " what is Dbeaver for" just brings up more questions than answers.

In a nutshell I know it is an SQL client, after that I am very lost.
 
What is the difference between Dbeaver and say a DBMS (Oracle, MySQL, etc.)

**Comment by p91paul** [comment, marked as answer]
DBMS like Oracle or MySQL are server side applications. Their task is to organize data as tables with indexes, and allow to query them. They do not have a user interface, only APIs to be used by clients. By contrast, clients like dbeaver, or sqlplus, sql developer, phpmyadmin, mysql workbench, are able to call the DBMS and display the results in a user-friendly way.
The DBMS and the client are usually installed on different computers (the database on the server, the clients on the user PC's), but that's not a requirement.

There are some DBMS that instead of having a client/server architecture are just files on the user filesystem (SQLite, Access, H2). For this kind of systems, the difference is much thinner, since the code to handle tables and queries is in the client itself. You may refer to the file as "the database", and the application is the client.
